src/Files.App/ViewModels/Dialogs/DynamicViewModel.cs

```diff
@@ -264,12 +264,12 @@ public DynamicDialogButtons DynamicButtonsEnabled
 		/// </summary>
 		public Action HideDialog { get; set; }
 
-		private Action<DynamicDialogViewModel, ContentDialogButtonClickEventArgs> primaryButtonAction;
+		private Action<DynamicViewModel, ContentDialogButtonClickEventArgs> primaryButtonAction;
```

Comment:
Might be confusing when using multiple namespaces

Reply:
As well not include Settings, Properties, Page like AboutSettingsViewModel. This is not accepted(AboutViewModel will be okay).

Reply:
I'm not a fan of this approach. While I'd agree to shorten longer class names, in this example, the full name is actually necessary.

Someone looking at the code might question themselves: "DynamicViewModel of what? How is it dynamic? Oh, it's for a dialog..."

Reply:
Then how about Settings.AboutViewModel? I think we should avoid duplication because you can hover on them and look for where it from and what it represents if they have the XAML comment. That's why I changed Properties.PropertiesGeneral to Properties.GeneralPage(although Page is necessary, they didn't have it.